config/default.yml

```diff
@@ -288,6 +288,13 @@ Rails/LinkToBlank:
   Enabled: true
   VersionAdded: '0.62'
 
+Rails/NewGlobalModel:
+  Description: 'Disallows addition of new global models to `app/models`. Prefer Rails Engines or namespaces.'
+  Enabled: true
```

Comment:
I think this would need to be disabled by default, since is goes against the typical approach.

Reply:
Oh, good call. That was the default from the cop generation rake task. Fixed in https://github.com/rubocop-hq/rubocop-rails/pull/152/commits/70cebcc159cff2a4114daa4491ce326eb4f9573a